src/components/Edge.vue

```diff
@@ -88,6 +84,9 @@ export default {
         x: this.centerEnd.x - this.svgRect.x,
         y: this.centerEnd.y - this.svgRect.y
       }
+    },
+    bezierOffset () {
+      return 0.5 * (this.centerEnd.x - this.centerStart.x)
```

Comment:
I believe this equation is not desirable. If the endpoint of the bezier curve is to the left of the starting point, the line will point into the material card. It should, however, always start off by going to the right.

Reply:
That's a rather splendid point. I'll get to it right away!

Reply:
No. You see, the SVG only has a constant amount of padding, and, as such, I'm worried that the curve may bleed over the edge. It might be possible to adjust the padding dynamically, but perhaps we should choose an entirely different approach.

Reply:
Oh, i see. Yet this does seem to be an issue with the current approach in general. I was able to reproduce the padding issue with a constant curvature ![here](https://user-images.githubusercontent.com/31693992/69764500-ca85d800-1170-11ea-8d96-8d71d697087f.png).
I understand your concern, and thus any dynamic curvature would reduce schlafability by some amount whilst increasing anxiety in demonstration situations.

We could revert back to only generating the contained `<path>` nodes and `v-for`ing them on a big svg the size of the screen.

Reply:
Assuming correctness of our current approach, I'd like to still pledge for the solution I proposed (including an abs), since it does certainly look quite nice might i say. 
![see here](https://user-images.githubusercontent.com/31693992/69764760-c8704900-1171-11ea-82fd-1ca2b3ce4781.png)

Reply:
A constant curvature value should not produce this error if the padding is at least a few pixels larger than the curvature, right? A global SVG seems unschlafable and may further complicate infinite scrolling.

Reply:
Correct. Choosing appropriate values for curvature and padding will prevent running into this issue.
However, as the original intent implies, constant curvature is also unschlafable because it will only look good for very specific distances (and especially not small ones).
I personally conclude that both approaches are innappropriate and a major redesign needs to take place.

Reply:
I would like to suggest a third approach: Using a constant curvature, which is reduced in special scenarios where the start end end position are close to each other.

Reply:
Sounds like a step in the right direction. We could also use a dynamic function like the one i suggested, but have it cap out hyperbolically at the padding value.

Reply:
Yes, that sounds reasonable. I hearby complete this round tableing session.